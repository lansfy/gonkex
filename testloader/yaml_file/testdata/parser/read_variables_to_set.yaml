- name: loader MUST read tests with variables_to_set section
  method: POST
  path: /some/path
  request: requestbody
  response:
    200: body200
    300: body300
    400: body400
  variables_to_set:
    200:
      var1: "path21"
      var22: "path22"
    300:
      var1: "path31"
      var33: "path33"
    400:
      var1: "path41"
      var44: "path44"
