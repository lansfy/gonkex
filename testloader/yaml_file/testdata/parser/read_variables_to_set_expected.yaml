- GetName: loader MUST read tests with variables_to_set section
  GetMethod: POST
  Path: /some/path
  GetRequest: requestbody
  GetResponses:
    200: body200
    300: body300
    400: body400
  GetVariablesToSet:
    200:
      var1: "path21"
      var22: "path22"
    300:
      var1: "path31"
      var33: "path33"
    400:
      var1: "path41"
      var44: "path44"
  GetFileName: testdata/parser/read_variables_to_set.yaml
  FirstTestInFile: true
  LastTestInFile: true
